pkg/apis/migration/v1alpha1/migcluster_types.go

```diff
@@ -218,6 +220,21 @@ func (m *MigCluster) GetClusterSubdomain(c k8sclient.Client) (string, error) {
 	return clusterSubdomain, nil
 }
 
+// GetOperatorVersion retrieves the operator version from the respective controllers ConfigMap
+func (m *MigCluster) GetOperatorVersion(c k8sclient.Client) (string, error) {
+	clusterConfig, err := m.GetClusterConfigMap(c)
+	if err != nil {
+		return "", liberr.Wrap(err)
+	}
+
+	operatorVersion, ok := clusterConfig.Data[OperatorVersionKey]
+	if !ok {
+		return "", liberr.Wrap(err)
```

Comment:
@ibolton336 I don't think this line isn't doing what we want, since no `err` is generated it's actually going to return the err (or nil value) from line 225. 

We need to create an error to return, like 

```
return "", liberr.Wrap(errors.Errorf("configmap key %v not found in configmap %v/%v" for migcluster %v/%v, 
OperatorVersionKey,  clusterConfig.Namespace, clusterConfig.Name, m.Namespace, m.Name))
```